packages/esm-service-queues-app/src/active-visits/active-visits-table.resource.ts

```diff
@@ -158,9 +156,7 @@ export const mapVisitQueueEntryProperties = (
   status: queueEntry.status.display as QueueStatus,
   statusUuid: queueEntry.status.uuid,
   waitTime: queueEntry.startedAt ? `${dayjs().diff(dayjs(queueEntry.startedAt), 'minutes')}` : '--',
-  visitStartDateTime: queueEntry.startedAt,
```

Comment:
Note:  This mapping is actually the root of the problem.

Reply:
Does this code change the visit information in the database? I don't have the full context but it appears to be constructing data for queue entries.

Reply:
I had not seen the full code. This is OK with me.